hw/ip/trial1/data/trial1_sec_cm_testplan.hjson

```diff
@@ -0,0 +1,16 @@
+// Copyright lowRISC contributors.
```

Comment:
Do we need this file?

Reply:
Same comment as earlier - if we dont check it in, it will show up as untracked file which would be annoying.

Reply:
Gee, I never noticed there was a trial1 ip :)